Le_duong/Laravel/eCommerce/app/Http/Controllers/ProductDetailsController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use DB;
+class ProductDetailsController extends Controller
+{
+    public function show ($id) {
```

Comment:
```suggestion
    public function show ($id)
    {
```
lúc để trên lúc để dưới thế nhỉ

Reply:
xong e xửa a